content/uk/2-1-Identifiers.md

```diff
@@ -1,7 +1,31 @@
 ## 2.1. Значення, ідентифікатор, змінна та константа, літерал, присвоєння
 
+> Ідентифікатор (identifier)
+
 Найважливіше у програмуванні — це давати зрозумілі імена ідентифікаторам (змінним, константам, параметрам, функціям, класам і т.д.) і розташовувати їх в програмі так, щоб вони були видні у потрібних місцях. При цьому моємо намагатися скоротити їх область видимості. Ідентифікатори можуть бути глобальними, оголошеними у файлі або імпортованими з інших модулів програми.
 
+Є різні конвенції (домовленості) про іменування, традиції мов і парадигм, але у всіх них імена мають бути _консистентні_ (однотипні) і _зрозумілі_ навіть для тих, хто цей код не писав. Приклади хороших імен ідентифікаторів:
+
+- Глобальні змінні: `MAX_PURCHASE`, `REQUEST_TIMEOUT`, `ERROR_CODES`
+- Колекції: `prices`, `goods`, `boughtItems`
+- Окремі значення: `amount`, `groupName`, `total`
+- Функції: `calculateSubtotal`, `calculateTotal`, `validateExpenses`
```

Comment:
@tshemsedinov I believe we are missing `;` (semi-colon) at the end of every example, aren't we? Should I add semi-colon both to UA/RU versions?

```suggestion
- Глобальні змінні: `MAX_PURCHASE`, `REQUEST_TIMEOUT`, `ERROR_CODES`;
- Колекції: `prices`, `goods`, `boughtItems`;
- Окремі значення: `amount`, `groupName`, `total`;
- Функції: `calculateSubtotal`, `calculateTotal`, `validateExpenses`;
```